### PR TITLE
Pull in sqipu html2pdf updates

### PR DESCRIPTION
**Pull request by rossanderson75**
Merges in updates from the squipu/html2pdf master repro to match version 5.2.7

### CODE REVIEW

**Review by elsbury13 — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -17,5 +17,6 @@
 /bin/
 /vendor/
 /composer.lock
+/composer.phar
```
Comment (kyam-mayden) [inline .gitignore]: How come we've added this line?

File: phpunit.xml [hunk 2]
```diff
@@ -6,10 +6,24 @@
...
+            <exclude>
+                <directory>src/Tests</directory>
+                <directory>src/config</directory>
+            </exclude>
+        </whitelist>
+    </filter>
+    <!-- logging>
+        <log type="coverage-html" target="build/logs/coverage" charset="UTF-8" yui="true" highlight="true"/>
+        <log type="coverage-clover" target="build/logs/clover.xml"/>
+    </logging -->
```
Comment (kyam-mayden) [inline phpunit.xml]: M: Do we need this, or can we delete entirely?

**Review by kyam-mayden — CHANGES_REQUESTED** [review]

File: phpunit.xml (hunk 2 above)
Comment (elsbury13) [reply phpunit.xml]: Dont think we need this, Ive checked a few other phpunit files in other projects and we dont use this
[692ae07](https://github.com/may-den/html2pdf/pull/3/commits/692ae0768ae548a9f0ac661b4eaf26abce9f8df9)

File: .gitignore (hunk 1 above)
Comment (elsbury13) [reply .gitignore]: Removed as this file is not there [692ae07](https://github.com/may-den/html2pdf/pull/3/commits/692ae0768ae548a9f0ac661b4eaf26abce9f8df9)

File: composer.json [hunk 3]
```diff
@@ -19,7 +19,7 @@
         "tecnickcom/tcpdf": "^6.3"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.5",
+        "phpunit/phpunit": "^5.0 || ^9.0",
```
Comment (liamjcrewe) [inline composer.json]: 🟠  Are we sure this shouldn't just be `^9.0`? I know this came from the main repo but our fork didn't have v5.0 support before

File: phpunit.xml [hunk 4]
```diff
@@ -1,14 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-     xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/8.5/phpunit.xsd"
-     bootstrap="./vendor/autoload.php" colors="true" backupGlobals="false"
+     xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.1/phpunit.xsd"
```
Comment (liamjcrewe) [inline phpunit.xml]: 🟠 How come we downgraded from 8.5 to 4.1?

**Review by liamjcrewe — CHANGES_REQUESTED** [review]

File: composer.json (hunk 3 above)
Comment (elsbury13) [reply composer.json]: Yeah we dont need this, it looks like its only there to support 5.6 in there main. Updated to 9.6 latest for PHP7.4 support
[7edccbb](https://github.com/may-den/html2pdf/pull/3/commits/7edccbb26ee4b0858d71fd3ac0210db11b33e141)

File: phpunit.xml (hunk 4 above)
Comment (elsbury13) [reply phpunit.xml]: 🤷 changed it to match the PHPUnit version
[7edccbb](https://github.com/may-den/html2pdf/pull/3/commits/7edccbb26ee4b0858d71fd3ac0210db11b33e141)

**Review by liamjcrewe — APPROVED** [review]

**Review by kyam-mayden — APPROVED** [review]

**Pull request merged**